### ISSUE TITLE
Bug: Error "Domain not allowed" for substack.com

### ISSUE THREAD

**Issue by joedavison** [bookmark-org/bookmark#90]
<img width="968" alt="Screenshot 2023-06-21 at 1 50 07 PM" src="https://github.com/bookmark-org/bookmark/assets/2687550/0dee2217-2033-406a-a5d8-c93bda577572">

observe: 

substack.com is not present in /priv/static/blocked_domains.txt

archive a link to substack page, e.g. https://whyisthisinteresting.substack.com/p/the-platinum-photography-edition

application error messages indicates domain not allowed

**Comment by leobz** [comment]
Yes, the problem was `ack.com` domain inside the `blocked_domains.txt`. I also had issues with pages that had the `fandom.com` subdomain also included there.

The real problem was here
```
 blocked_domains
    |> Enum.find(&String.contains?(url, &1))
```
    
I solved the problem in this PR https://github.com/bookmark-org/bookmark/pull/100
